src/main/kotlin/com/pubnub/api/models/consumer/history/PNFetchMessage.kt

```diff
@@ -38,7 +38,8 @@ data class PNFetchMessageItem(
     val meta: JsonElement?,
     val timetoken: Long,
     val actions: Map<String, Map<String, List<Action>>>? = null,
-    val messageType: HistoryMessageType?
+    val messageType: HistoryMessageType?,
+    val error: String? = null
```

Comment:
I'm not a big fan of adding an `error` field here, using `PNStatus` would seem like a better option for communicating this, however it wouldn't work in the `.sync()` case I think since there's no PNStatus returned, only `Output` or `Exception`... 

If we don't have other options, can we at least not make this a String, and make it an enum instead, with just one possible value (or null) for now? we can add more in the future if we come up with any other possible errors for this

Reply:
I can create entry in PubNubError enum.